core/src/main/java/com/mapzen/android/graphics/model/MarkerOptions.java

```diff
@@ -5,6 +5,7 @@
 
 import android.graphics.drawable.Drawable;
 
+
```

Comment:
Is this class only used for `BitmapMarkers` or also for `Markers` (backed by `MapData`)? If so, we should clarify the naming.

Reply:
Updated to `BitmapMarkerOptions` and deprecated `MarkerOptions` !